scss/_navbar.scss

```
@@ -1,29 +1,36 @@
-// Contents
-//
-// Navbar
-// Navbar brand
-// Navbar nav
-// Navbar text
-// Responsive navbar
-// Navbar position
-// Navbar themes
-
-
 // Navbar
 //
 // Provide a static navbar from which we expand to create full-width, fixed, and
 // other navbar variations.
 
 .navbar {
+  // scss-docs-start navbar-css-vars
+  --#{$variable-prefix}navbar-padding: #{$navbar-padding-y} #{if($navbar-padding-x == null, 0, $navbar-padding-x)};
```

Please separate paddings for X and Y as it would allow us to use them separately in vertical and horizontal calculations. Just like these 2: `--#{$variable-prefix}navbar-toggler-padding-y` and `--#{$variable-prefix}navbar-toggler-padding-x`

---

I tend to agree with that, could ease quite a few things.
```suggestion
  --#{$variable-prefix}navbar-padding-x: #{if($navbar-padding-x == null, 0, $navbar-padding-x)};
  --#{$variable-prefix}navbar-padding-y: #{$navbar-padding-y};
  --#{$variable-prefix}navbar-padding: var(--#{$variable-prefix}navbar-padding-y) var(--#{$variable-prefix}navbar-padding-x);
```